sdk/storage/Azure.Storage.Blobs/tests/AppendBlobClientTests.cs

```diff
@@ -128,11 +128,14 @@ public async Task CreateAsync_CPK()
                 // Arrange
                 var blobName = GetNewBlobName();
                 AppendBlobClient blob = InstrumentClient(container.GetAppendBlobClient(blobName));
-                blob = InstrumentClient(new AppendBlobClient(GetHttpsUri(blob.Uri), blob.Pipeline));
                 CustomerProvidedKey customerProvidedKey = GetCustomerProvidedKey();
+                blob = InstrumentClient(new AppendBlobClient(
```

Comment:
Nit - in the future it might be slightly less work to write a `WithCustomerProvidedKey(key)` extension method instead of making this change everywhere.   I don't think you need to make that change now though.

Reply:
Agreed, I was thinking about adding this when I was updating the unit tests.

Reply:
I was talking about adding this as an extension method for just the tests.

Does this make sense as a public API on the clients?  What's the mainline scenario for switching between using the encryption key and not without the customer reuploading the blob?  This feels like a confusing API to me.e